### PR TITLE
Add note & edit note

### PR DESCRIPTION
**Pull request by aayushjain-code**

### CODE REVIEW

**Comment by aayushjain-code** [comment]
Live Preview: https://dancing-banoffee-3f6504.netlify.app/

File: src/Components/EditCard/EditCard.jsx [hunk 1]
```diff
@@ -0,0 +1,121 @@
...
+		priority: editItem.priority,
+		selectedBackgroundColor: editItem.selectedBackgroundColor,
+	});
+
+
+
+	return (
+
+		<div className="modal-edit flex flex-align-center flex-justify-center">
+			{console.log("ddd", edit.editItem)}
```
Comment (ShekharDhangar) [inline src/Components/EditCard/EditCard.jsx:24]: can remove console.log

File: src/Components/EditCard/EditCard.jsx [hunk 2]
```diff
@@ -0,0 +1,121 @@
...
+						>
+							<option value="Label" hidden>
+								Label
+							</option>
+							<option value="Home">Home</option>
+							<option value="Work">Work</option>
+							<option value="Personal">Personal</option>
+							<option value="Exercise">Exercise</option>
+							<option value="Chores">Chores</option>
+							<option value="Health">Health</option>
```
Comment (ShekharDhangar) [inline src/Components/EditCard/EditCard.jsx:87]: you could store this in array and then map over it, in this way you will have more control over it

File: src/Context/noteContext.js [hunk 3]
```diff
@@ -0,0 +1,104 @@
...
+
+
+const NoteContext = createContext();
+
+const NoteProvider = ({ children }) => {
+
+	const { authDispatch } = useAuth();
+	const config = {
+		headers: {
+			authorization: localStorage.getItem("tokenNotesApp")
```
Comment (ShekharDhangar) [inline src/Context/noteContext.js:13]: you are using localstorage many times so you could build a customm hook  for it and use it

**Review by ShekharDhangar — APPROVED** [review]
Look at the suggested changes, rest looks good

File: src/Components/ColorPalette/ColorPalette.jsx [hunk 4]
```diff
@@ -0,0 +1,55 @@
...
+	"#238AC5",
+];
+
+
+export const ColorPalette = ({ setInputCardDetails, inputCardDetails }) => {
+	const [isHidden, setHidden] = useState(true);
+	return (
+		<>
+			<div
+				onMouseOver={() => {
```
Comment (MuskanChhatrasal) [inline src/Components/ColorPalette/ColorPalette.jsx:25]: Try to format the code, can use Prettier for that.

File: src/Components/Header/Navbar.jsx [hunk 5]
```diff
@@ -1,13 +1,13 @@
...
+import { useAuth } from '../../Context/authContext'
 
 const NavBar = () => {
 	const { logout } = useAuth();
 	return (
 		<header class="header">
-			<Link to="/" className='logo'> <i class="fa-solid fa-book-open-reader"></i>  Fresh-Keep</Link>
+			<Link to="/notes" className='logo'> <i class="fa-solid fa-book-open-reader"></i>  Fresh-Keep</Link>
 
 			{/* <form action="" class="search-form">
```
Comment (MuskanChhatrasal) [inline src/Components/Header/Navbar.jsx:12]: Remove the commented code.

**Review by MuskanChhatrasal — APPROVED** [review]
Overall great work. Just a few suggestions - try to minimize the use of inline styling, remove commented code and format the code it will make code more readable. Else good to merge.

File: src/Context/noteContext.js (hunk 3 above)
Comment (aayushjain-code) [reply src/Context/noteContext.js:13]: will do it for sure

File: src/Components/ColorPalette/ColorPalette.jsx (hunk 4 above)
Comment (aayushjain-code) [reply src/Components/ColorPalette/ColorPalette.jsx:25]: Sure will do it for sure ;-)

**Pull request merged**